src/blocks/image.v1.yaml

```diff
@@ -21,14 +21,8 @@ definitions:
         allOf:
           - $ref: ../misc/asset.v1.yaml
           - properties:
-                alt:
-                    title: Alternative text
-                    type: string
-                uri:
-                    title: Image URI
-                    type: string
-                    format: uri
-                    pattern: ^https://
+                image:
```

Comment:
This is a proposal based on a discussion yesterday, coordination is needed on deploying and we shouldn't do that yet.

We spoke about generating static files in the CDN in the IIIF URI syntax for the sizes that we require, which would allow us to generate all the images ahead of time, switch over, then later down the line use a proper IIIF server.